### PR TITLE
PEX: Documentation on how a Policy maps Constraint Field IDs

### PR DESCRIPTION
**Pull request by reinkrul**
For the token introspection response.

Also fixed styling issues (2 spaces instead of tabs to keep code blocks a bit more condensed).

### CODE REVIEW

File: docs/pages/deployment/pex.rst [hunk 1]
```diff
@@ -22,71 +22,82 @@ To use file-based configuration, you need to define the path to a directory that
...
+  }
+
+Where ``example_scope`` is the scope that the presentation definition is associated with.
+The ``presentation_definition`` object contains the presentation definition that should be used for the given scope.
+The ``wallet_owner_type`` field is used to determine the audience type of the presentation definition, valid values are ``organization`` and ``user``.
+
+OAuth2 Token Introspection field mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The fields that contain an ``id`` property (e.g., ``example_credential_type`` in the example above) are returned in the OAuth2 Token Introspection response.
```
Comment (gerardsn) [inline docs/pages/deployment/pex.rst:87]: `field(s)` is a very generic term. I think it should be very clear here that this refers to the `input_descriptor.contraints.field.id`s, and only these id's.

File: docs/pages/deployment/pex.rst [hunk 2]
```diff
@@ -22,71 +22,82 @@ To use file-based configuration, you need to define the path to a directory that
...
+The value of the Verifiable Credential that the matched field constraint are included in the response as claims.
```
Comment (gerardsn) [inline docs/pages/deployment/pex.rst:88]: I don't know what this means

File: docs/pages/deployment/pex.rst [hunk 3]
```diff
@@ -22,71 +22,82 @@ To use file-based configuration, you need to define the path to a directory that
...
+Writer of policies should take into consideration:
+- fields that are intended to be used for logging or authorization decisions should have a distinct identifier.
+- claims ideally map a registered claim name.
```
Comment (gerardsn) [inline docs/pages/deployment/pex.rst:91]: registered here: https://www.iana.org/assignments/jwt/jwt.xhtml#claims?

File: docs/pages/deployment/pex.rst [hunk 4]
```diff
@@ -22,71 +22,82 @@ To use file-based configuration, you need to define the path to a directory that
...
+- overwriting properties already defined in the token introspection endpoint response is forbidden.
```
Comment (gerardsn) [inline docs/pages/deployment/pex.rst:92]: Add link to normative source; RFC021 or the OAPI spec?

**Review by gerardsn — CHANGES_REQUESTED** [review]

File: docs/pages/deployment/pex.rst (hunk 3 above)
Comment (gerardsn) [reply docs/pages/deployment/pex.rst:91]: Or just reference IANA registries in general? As long as it is not a reference to "some" registry

Comment (reinkrul) [reply docs/pages/deployment/pex.rst:91]: "regsitered" is a broad term here i.m.o., but IANA is a good choice.

File: docs/pages/deployment/pex.rst [hunk 5]
```diff
@@ -22,71 +22,84 @@ To use file-based configuration, you need to define the path to a directory that
...
+Where ``example_scope`` is the scope that the presentation definition is associated with.
+The ``presentation_definition`` object contains the presentation definition that should be used for the given scope.
+The ``wallet_owner_type`` field is used to determine the audience type of the presentation definition, valid values are ``organization`` and ``user``.
+
+OAuth2 Token Introspection field mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The input descriptor constraint fields that contain an ``id`` property (``input_descriptor.contraints.field.id``) are returned in the OAuth2 Token Introspection response.
+The value of the Verifiable Credential that the matched field is included in the response as claims.
+E.g., in the example above, a claim named ``example_credential_type`` is added to the introspection response, containing the Verifiable Credential ``type`` property.
```
Comment (woutslakhorst) [inline docs/pages/deployment/pex.rst:89]: You mean the value of the `type` property. An example of an introspection response would be really helpful here.

**Review by woutslakhorst — CHANGES_REQUESTED** [review]

File: docs/pages/deployment/pex.rst [hunk 6]
```diff
@@ -86,7 +92,18 @@ OAuth2 Token Introspection field mapping
 
 The input descriptor constraint fields that contain an ``id`` property (``input_descriptor.contraints.field.id``) are returned in the OAuth2 Token Introspection response.
 The value of the Verifiable Credential that the matched field is included in the response as claims.
-E.g., in the example above, a claim named ``example_credential_type`` is added to the introspection response, containing the Verifiable Credential ``type`` property.
+E.g., in the example above, a claim named ``fullName`` is added to the introspection response, containing the Verifiable Credential ``type`` property.
```
Comment (woutslakhorst) [inline docs/pages/deployment/pex.rst:95]: It doesn't contain the `type` property?

**Review by woutslakhorst — CHANGES_REQUESTED** [review]

**Review by gerardsn — APPROVED** [review]

**Pull request merged**